fees/stader/index.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  Adapter,
+  Fetch,
+  FetchOptions,
+  FetchResultFees,
+} from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { queryDune } from "../../helpers/dune";
+
+type IMainnetResponse = {
+  month: string;
+  monthly_operator_reward: number;
+  monthly_stader_revenue: number;
+  monthly_user_reward: number;
+};
+
+type IAltResponse = {
+  month: string;
+  monthly_stader_revenue: number;
+  monthly_supply_side_fee: number;
+};
+
+const DUNE_QUERY: { [key: string]: string } = {
+  ethereum: "3376313",
+  polygon: "3376354",
+  bsc: "3376374",
+};
+
+const formatDay = (date: string): string => date.split(" ")[0];
+
+const formatTimestampToISO = (timestamp: number | string): string => {
+  const date = new Date(Number(timestamp) * 1000);
+  return date.toISOString().split("T")[0];
+};
+
+const fetchOnEthereum = async (
+  timestamp: number,
+  _: any,
+  {}: FetchOptions,
+  query: string
+): Promise<FetchResultFees> => {
+  const isoTimestamp = formatTimestampToISO(timestamp);
+  const rawResult: IMainnetResponse[] = await queryDune(query);
+  const results = rawResult.map((r) => ({ ...r, month: formatDay(r.month) }));
+
+  const relevantResults: IMainnetResponse | undefined = results.find(
+    ({ month }) => { return month === isoTimestamp });
+
+  if (!relevantResults) return { timestamp, dailyFees: 0, dailyRevenue: 0 };
+  const { monthly_operator_reward, monthly_stader_revenue, monthly_user_reward } = relevantResults;
+  const monthlyRevenues = monthly_stader_revenue;
+  const monthlyFees = monthly_operator_reward + monthly_stader_revenue + monthly_user_reward;
+
+  return {
+    timestamp,
+    dailyFees: monthlyFees / 30,
```

Comment:
i think div / 30 is not correct value

Reply:
Data query was like :

![image](https://github.com/user-attachments/assets/db4dba2c-fb20-40d6-80cf-6b686cd81712)

I proceeded as follows: 
- The fees collected by the protocol through ether staking include all the rewards accumulated by the various participants (users, operator, protocol)
- The revenues are only the rewards that are not redistributed
- Since the reports are published monthly, I divided the values of each report by 30 to convert them to daily